vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -17,15 +17,35 @@
     <v-expand-transition>
       <div v-if="!editing" class="address-block">
         <div class="address-block__info pre-line">
-          <div class="address-block__info-row">{{ addressLocal.streetAddress }}</div>
-          <div class="address-block__info-row">{{ addressLocal.streetAddressAdditional }}</div>
+          <div class="address-block__info-row street-address">
```

Comment:
It seemed silly to add a classname for delivery instructions (eg, to select it externally and italicize it) without adding classnames to all other fields.